### PR TITLE
Add the show subcommand and docs for showing a site

### PR DESCRIPTION
**Pull request by ejcx**
let me know what you think @shurcooL

### CODE REVIEW

**Comment by shurcooL** [comment]
> This "no command" was copied from https://www.passwordstore.org/

I see.

I think this is good, but I would consider going one step further and removing the previous way to retrieve passwords.

As is, after this PR, it'll still be possible to get passwords the old way which is susceptible to shadowing issues, and new people may not know which of the two ways they should use - or why 2 ways to do the same thing exist.

> passgo stores, retrieves, generates, and synchronizes passwords securely and is written in Go!

Since the description even mentions that passgo is written in Go, I think it'd be nice to embrace Go style and make the retrieval command explicit, and remove the more "magical" no command way.

As a result, the documentation becomes simpler and shorter, and there's less confusion about ways to show the password.

I know it'd be a breaking API change, but I suggest you consider it anyway. People who are used to the current way of showing passwords will see a helpful error message and understand that `passgo site` doesn't work anymore, and hopefully getting used to `passgo show site` is going to be easy. Or, you could consider deprecating the old style and removing it later.

**Comment by ejcx** [comment]
I think `passgo show` is good and reflects the go ethos better, like you mentioned. I'll just go ahead and remove the other way.

**Comment by ejcx** [comment]
What do you think about this?

**Comment by shurcooL** [comment]
Looks great! Are you planning to update the README as part of this PR?

**Comment by ejcx** [comment]
Yes. I'll do that and the merge. Thanks for the reminder.

File: README.md [hunk 1]
```diff
@@ -65,6 +65,13 @@ Here we are adding mint.com to the password store, but more specifically to the
 ```
 $ passgo insert mney/mint.com
 ```
```
Comment (shurcooL) [inline README.md]: For consistency, there should be a blank line separating the blocks here.

File: README.md [hunk 2]
```diff
@@ -65,6 +65,13 @@ Here we are adding mint.com to the password store, but more specifically to the
...
+#### passgo show group/pass-name
+Show is used to display a password in standard out. *Previously it was possible to display a password using `passgo group/pass-name` but this is no longer supported*.
```
Comment (shurcooL) [inline README.md]: And after this line.

Comment (shurcooL) [inline README.md]: I think you missed this comment.

It's very minor though.

Comment (ejcx) [inline README.md]: I believe I got it, didn't I?

https://github.com/ejcx/passgo/commit/8e4efffbb43a07d5d161e3745a2d17a16565023a#diff-04c6e90faac2675aa89e2176d2eec7d8R68

Comment (shurcooL) [inline README.md]: That's what my other comment was about. This one is about the missing blank line between lines 70 and 71.

Comment (ejcx) [inline README.md]: Done

**Pull request merged**